lib/plurimath/asciimath_parser/sin.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Plurimath
+  module AsciimathParser
+    # Sin Class
+    class Sin
```

Comment:
@suleman-uzair `Sin`, `Cos`, `Tan` are functions. 
So, I think they should be inside `Plurimath::AsciimathParser::Functions` ?

Reply:
Agree.